.gitignore

```diff
@@ -3,3 +3,4 @@ dist/
 node_modules/
 .DS_Store
 .env
+.expo
```

Comment:
We will not be using expo, right? We can remove this

Reply:
I did remove it. But I will double check again